debian8/product.yml

```diff
@@ -4,6 +4,6 @@ benchmark_root: "./guide"
 
 profiles_root: "./profiles"
 
-pkg_system: "dpkg"
+{{{ declare_pkg_system_and_manager(pkg_manager="apt-get") }}}
```

Comment:
Why was this change made? We need to keep a standard schema of `key: jinja_template`.

Reply:
+1

Reply:
IMO it's cleaner to just have pkg_system and pkg_manager as separate keys instead of this super long macro name and logic. It's way more self documenting to just have 2 keys

Reply:
You have missed the reasoning in the updated PR description. Technically, it is OK to violate the DRY principle by saying that if a system uses `yum`, it is RPM-based though - each approach has a weakness.